jobs/BnRStatusPollerJob.js

```diff
@@ -66,35 +66,55 @@ class BnRStatusPollerJob extends BaseJob {
     const plan = catalog.getPlan(instanceInfo.plan_id);
     return Promise
       .try(() => {
-        logger.info(`Check if the backup resouce ${backup_guid} and deployment resource ${instance_guid} already exists`);
+        logger.info(`Check if the backup resource ${backup_guid} and deployment resource ${instance_guid} already exists`);
         // For transition of sf to sfv2 we need to create bakcup and deployment resource
         // if they are not present
         // Can be removed once SFv2 transition is over
         return Promise
-          .try(() => eventmesh.apiServerClient.getResource(
-            CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT,
-            CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
-            instance_guid))
+          .try(() => eventmesh.apiServerClient.getResource({
+            resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT,
+            resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
+            resourceId: instance_guid
+          }))
           .catch(() => {
```

Comment:
shouldnt this be specificially for `NotFound` error.

Reply:
Yes. Corrected.